src/wirecloud/platform/static/js/wirecloud/Wiring.js

```diff
@@ -185,6 +185,21 @@
                 privates.get(this).connections.push(connection);
             }
 
+            // Create the conection on the server
```

Comment:
- Missing tests (see `js_tests/wirecloud/WiringSpec.js` for more details).
- This code make requests for persisting volatile connections when it shouldn't.
- The `commit` option must disappear and the code should assume `commit` === `true`.
- The `createConnection` method should return Promises.